components/desktop/sxhkd/patches/01-fix-usr-local.patch

```diff
@@ -0,0 +1,11 @@
+--- sxhkd-0.6.2/Makefile.orig	2020-08-02 00:53:02.000000000 +0000
++++ sxhkd-0.6.2/Makefile	2022-05-23 07:45:39.374284976 +0000
+@@ -7,7 +7,7 @@
+ LDFLAGS  ?=
+ LDLIBS    = $(LDFLAGS) -lxcb -lxcb-keysyms
+ 
+-PREFIX    ?= /usr/local
++PREFIX    ?= /usr
+ BINPREFIX ?= $(PREFIX)/bin
```

Comment:
I'd prefer a solution inside our Makefile (setting the PREFIX there as the use of ?= indicates that this is intended).
The reason for this is that your patch needs adoption every time the Makefile changes near to your patch while setting PREFIX in out Makefile would continue to work.